amf-client/shared/src/test/resources/production/tccs-crm-cases-sys-api/tccs-crm-system-api-v1.raml

```diff
@@ -0,0 +1,72 @@
+#%RAML 1.0
+baseUri: https://anypoint.mulesoft.com/mocking/api/v1/links/324e9f06-42e9-412e-8955-eace3920af23/ # 
```

Comment:
Should we rename this api and some of its content? does not seem public.

Reply:
@pope1838 do you know where this API came from? according to the git history this was added by us, but I can't see why we would do such thing (seems a bit arbitrary)